### ISSUE TITLE
VS 更新后编译不通过了：CS8983 A 'struct' with field initializers must include an explicitly declared constructor.

### ISSUE THREAD

**Issue by cuiliang** [NaBian/HandyControl#1122]
![image](https://user-images.githubusercontent.com/1972649/154779675-f4697b8e-0b91-4210-9d0e-272d8380ae4e.png)

参考：
- https://github.com/dotnet/sdk/issues/23971
- https://stackoverflow.com/questions/71145085/vs-2022-17-1-wont-build-c-sharp-10-struct-cs8983-17-0-5-liked-it-what-happen

**Comment by Yopler** [comment]
Hi 🖐,

This is in any case a HandyControl error (Not supposed to be in HandyControl issues).
If you read the answer to your stackoverflow link, I seem to be an error comming from your upgrade to C# 10. This error occur because you have a default `length `value
One of the solution proposed by stackoverflow is to create an empty constructor for your `struct`.
```cs
internal struct WINDOWPLACEMENT
{
public int length = Marsha;
public int flags;
// your other fields
public WINDOWPLACEMENT() { } //empty constructor that should allowed you to rebuild
}
```
hope it help.

**Comment by cuiliang** [comment]
@Yopler thanks， I have already solved it, see #1123

**Comment by Yopler** [comment]
Nice ! Sorry, I didn't see it !
